dist/source/repo-creator/choose-name.ts

```diff
@@ -2,30 +2,43 @@ import { autoinject } from 'aurelia-dependency-injection';
 import { computedFrom } from 'aurelia-binding';
 import { Router } from 'aurelia-router';
 import { EventAggregator } from 'aurelia-event-aggregator';
+import { Validation } from 'aurelia-validation';
 
 @autoinject()
 export class ChooseName {
 	constructor(
 		private router: Router,
-		private eventAggregator: EventAggregator
-	) {}
-	
+		private eventAggregator: EventAggregator,
+		validation: Validation
+	) {
+
+		this.validation = validation.on(this)
+			.ensure('newRepoName')
+			.isNotEmpty();
+	}
+
 	private templateOwner: string;
 	private templateName: string;
 
 	protected newRepoName: string = '';
+	protected validation: Validation;
```

Comment:
Can you remove this line and put `protected` on the constructor parameter?  This is a shortcut syntax in TypeScript that makes it so you don't have to do the assignment as the compiler does it for you.
```ts
export class ChooseName {
	constructor(
		private router: Router,
		private eventAggregator: EventAggregator,
		protected validation: Validation
	) {
		this.validation.on(this)
			.ensure('newRepoName')
			.isNotEmpty();
	}
}
```